Automattic/wp-calypso
# Fixes yarn version to 1.22.4 using yarn version policies

#### Changes proposed in this Pull Request

* Fixates yarn version to 1.22.4 usign [yarn policies](https://classic.yarnpkg.com/en/docs/cli/policies/)

#### Testing instructions

* Checkout the branch, delete `.yarn` and `.yarnrc` and run `yarn policies set-version 1.22.4`. There should be no differences with the files in this PR.

 * Checkout the branch, change your yarn version to something different than 1.22.4. Then run `yarn -v` inside the repo and check it reports `1.22.4`
